dias/analyzers/dataset_analyzer.py

```diff
@@ -80,7 +80,11 @@ def run(self):
         ds.get.index()
 
         # Use the tracker to get the chimestack files to analyze
-        cs_file_list = self.new_files(filetypes=self.instrument + "_corr")
+        end_time = datetime.datetime.utcnow() - self.offset
+
+        self.logger.info("Looking at files older than {0}".format(end_time))
```

Comment:
This was my debug log. Should we remove it?

Reply:
I like it! But we can make it a debug, instead